### ISSUE TITLE
🐛 function account_get_id set a local variable to user_id unexpectedly

### ISSUE THREAD

**Issue by ZhangGuangxu** [heroiclabs/nakama#270]
## Description

🐛  function account_get_id set a local variable to user_id unexpectedly

## Steps to Reproduce

my code is like this:

```
local nk = require("nakama")

local function update_wallet(user_id,changeset,reason,no_notif,account)
    local md = nil

    nk.logger_info("type(md) is "..tostring((type((md)) == "table" and "object" or type((md))))..", md is "..tostring(nk.json_encode(md)).."}")
    account = (account or nk.account_get_id(user_id))
    nk.logger_info("type(md) is "..tostring((type((md)) == "table" and "object" or type((md))))..", md is "..tostring(nk.json_encode(md)).."}")

  ...
end
```

and I invoke update_wallet(...) somewhere.

## Expected Result
local variable md should stay nil

## Actual Result

I am sure *nk.account_get_id(user_id)* is invoked.

this is the log content:
before *account = (account or nk.account_get_id(user_id))*
```
{"level":"info","ts":"2018-11-26T12:38:28.608+0800","msg":"type(md) is nil, md is null}","mid":"8add791d-8e78-4ed6-a8ca-0c4ed4703b88"}
```

and after *account = (account or nk.account_get_id(user_id))*
```
{"level":"info","ts":"2018-11-26T12:38:28.618+0800","msg":"type(md) is string, md is \"2bdfee20-0783-4e1e-845c-4c42fed2cf90\"}","mid":"8add791d-8e78-4ed6-a8ca-0c4ed4703b88"}
```
md is set to account.user_id.

## Context

client:  nakama-js

## Your Environment

- Nakama: 2.0.0+dev
   build nakama binary from commid(master branch): 613efe9bcab0fbbd931a2db8f2f8b445e7f70891
- Cockroach: 
- Environment name and version: {"runtime":"go1.11.1","cpu":4}
- Operating System and version: macOS 10.13.4  or Ubuntu 16.04.5 LTS

**Comment by zyro** [comment]
Confirmed this is an issue with the Lua runtime implementation used by Nakama. Tracked in https://github.com/yuin/gopher-lua/issues/216.

**Comment by zyro** [comment]
Fixed in dc77105bb7de50b3833a4d098033f38ca061f164 and will be part of the next release.
